proto/sdv/databroker/v1/types.proto

```diff
@@ -163,7 +151,49 @@ message Metadata {
   ChangeType change_type = 6;  // CONTINUOUS or STATIC or ON_CHANGE
   string description     = 7;
 
-  Allowed allowed = 10;
+  ValueRestriction value_restriction = 10;
```

Comment:
Do we really want to introduce it to this old API?

Reply:
If we don't add `min` and `max`, there is no way for users to know what is the correct range of values to publish, i.e databroker-perf (provider)

Reply:
Yes, but they have managed with that so far

Reply:
> Yes, but they have managed with that so far

I think before it worked since min, max were never checked for release 0.4.6, maybe even users never use min and max in the vss json file on databroker, however if we dont add this, we will not be able to generate correct values to measure the performance in case there are users using min max concept in their vss files.